### PR TITLE
Changes for r0.7.

### PR DESCRIPTION
**Pull request by avijit-nervana**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -165,61 +161,96 @@ the dependencies.
...
-lags the features and bug fixes available in the `master` branch of this repository.
+1. Install [bazel](#install-bazel-for-building-tensorflow-source) and other TensorFlow dependencies. Now, clone the source of the [tensorflow] repo to your machine. 
 
-### Running tests
+     :warning: You need the following version of TensorFlow: `v1.11.0-rc2`
 
-To run the C++ unit tests,
+        git clone https://github.com/tensorflow/tensorflow.git
+        cd tensorflow
+        git checkout v1.11.0-rc2
```
Comment (shresthamalik) [inline README.md]: Should we change this to v1.11 too? Similar to option 1.

File: README.md [hunk 2]
```diff
@@ -1,33 +1,57 @@
...
+
+        export PATH=$PATH:~/bin
+        source ~/.bashrc 
+
+### Option 1: Build nGraph using an existing TensorFlow installation
+
+1. Create and activate a [python virtual environment](#create-a-python-virtual-environment)
     
 2. Install TensorFlow v1.11.0. Note that this is a pre-release so you need 
    to use the following steps to install this:
```
Comment (shresthamalik) [inline README.md]: As v1.11.0 is official release. We may not need this line "Note that this is a pre-release so you need 
   to use the following steps to install this:"

File: README.md (hunk 1 above)
Comment (avijit-nervana) [reply README.md]: Yes - this should now be changed to r1.12 (but need to have the CI run with r1.12 first)

File: README.md (hunk 2 above)
Comment (avijit-nervana) [reply README.md]: Agree.

File: README.md (hunk 1 above)
Comment (shresthamalik) [reply README.md]: Ok. this one points to v1.11.0-rc2, should we change to v1.11.0 (remove the rc tag)?

Comment (avijit-nervana) [reply README.md]: `r1.12`

File: README.md [hunk 3]
```diff
@@ -149,12 +145,12 @@ The installation prerequisites are the same as described in the TensorFlow
         git checkout v0.6.1
```
Comment (shresthamalik) [inline README.md]: This is more of a comment from @sayantan-nervana , we were discussing this yesterday. Is their a constraint, to clone ngraph-tf in the *parent* directory of tensorflow repo, as mentioned in Step 6? If not, we can remove this part "one level above -- in the  *parent* directory of the `tensorflow` repo cloned in step 1:"

We might want to keep the ``` cd ..``` in the instructions to exit out of the tensorflow repo.

File: README.md [hunk 4]
```diff
@@ -38,8 +62,7 @@ a variety of nGraph-enabled backends: CPU, GPU, and custom silicon like the
 
         pip install -U tensorflow
 
-3. Checkout `v0.6.1` from the `ngraph-tf` repo and build the bridge
-   as follows: 
+3. Checkout `v0.6.1` from the `ngraph-tf` repo and build it:
```
Comment (sayantan-nervana) [inline README.md]: the PR name says `Changes for r0.7`, but we have `v0.6.1` in many places.
will we be updating the readme once during 0.7 release? in that case all the 0.6.1's need to be 7

File: README.md [hunk 5]
```diff
@@ -1,33 +1,57 @@
...
         source your_virtualenv/bin/activate # bash, sh, ksh, or zsh
+
+Typically the following python packages are also needed `numpy mock keras keras_applications`.
+
+        pip install -U numpy mock keras keras_applications
+
+Note: Depending on the version of Python and the packages already installed on your system,the above list may vary.
+
+#### Install bazel for building TensorFlow Source
+
```
Comment (sayantan-nervana) [inline README.md]: we could mention that this step is not needed for option 1

File: README.md [hunk 6]
```diff
@@ -1,33 +1,57 @@
...
-   systems with Python 3.n or Python 2.7, these commands are
+You need to instantiate a specific kind of `virtualenv`  to be able to proceed with the `ngraph-tf` installation. For Python 3.n or Python 2.7, do
 
         virtualenv --system-site-packages -p python3 your_virtualenv 
         virtualenv --system-site-packages -p /usr/bin/python2 your_virtualenv  
         source your_virtualenv/bin/activate # bash, sh, ksh, or zsh
+
+Typically the following python packages are also needed `numpy mock keras keras_applications`.
+
+        pip install -U numpy mock keras keras_applications
```
Comment (shresthamalik) [inline README.md]: From @kanvi-nervana  : "Add keras_preprocessing" to the list of python packages

**Pull request closed without merging**